src/pages/panel/views/main.js

```diff
@@ -53,17 +53,29 @@ function reloadTab(host, event) {
 }
 
 async function togglePause(host, event) {
+  host.alert = '';
+
   const { paused, pauseType } = event.target;
+  const { options, stats } = host;
 
-  host.alert = '';
+  if (paused) {
+    const pausedHostname = Object.keys(options.paused)
+      .sort()
+      .reverse()
+      .find((domain) => stats.hostname.endsWith(domain));
```

Comment:
why to sort and reverse if you do a find? pessimistically you have to iterate full list 3 times.

Reply:
We have to, as otherwise, if somehow a user adds `onet.pl` and `www.onet.pl`, and he is on `www.onet.pl`, we must remove the second entry from the paused list. However, it would be chosen if `onet.pl` is before.